init-jfr-test-framework.inc

```diff
@@ -4,8 +4,20 @@ set -e
 # Path to this source file, stored in BASH_SOURCE
 root_directory=$(dirname "${BASH_SOURCE[0]}")
 sources="$root_directory/src"
+sh_unit_directory="$root_directory/.shunit2"
 export DEFAULT_CWP_VERSION="1.5"
 
+if [ ! -d "$sh_unit_directory" ]
```

Comment:
The idea is: By default the framework is using the content of "sh_unit_directory" to locate the shUnit2 library. In case that directory is not a valid one (it does not exist or it is not a directory), it will read the arguments just to see if the location has been passed as argument (thinking always in  CWP or JFR repositories and their tests). If $1 is empty or $1 is not a valid directory, the script will show an error and exit.
But I repeat: the first location where the script will look for the library is the default expected one